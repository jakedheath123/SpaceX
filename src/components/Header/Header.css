.header-container {
  display: grid;
  grid-template-areas: "links links links";
  grid-gap: 1rem;
  margin-top: -2rem;
  margin: 0% 10% 0% 10%;
}

.header-logo {
  grid-area: links;
  display: flex;
  align-items: center;
  justify-content: flex-start;
}

.header-logo img {
  width: 150px;
  height: 150px;
}
