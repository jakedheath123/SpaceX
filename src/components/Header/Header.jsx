import React from "react";

import "./Header.css";
import logo from "../../images/logo.jpg";

const Header = () => {
  return (
    <header className="header-logo">
      <img src={logo} alt="spaceX logo" />
    </header>
  );
};

export default Header;
