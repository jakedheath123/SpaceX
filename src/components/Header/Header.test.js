import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

import Header from "./Header";

describe("Header component test suite", () => {
  it("renders without crashing", () => {
    shallow(<Header />);
  });
  it("renders Header component", () => {
    const component = renderer.create(shallow(<Header />)).toJSON();

    expect(component).toMatchSnapshot();
  });
});
