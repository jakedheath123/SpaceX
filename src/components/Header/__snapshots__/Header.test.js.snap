// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test suite renders Header component 1`] = `
<header
  className="header-logo"
>
  <img
    alt="spaceX logo"
    src="logo.jpg"
  />
</header>
`;
